include/casmutils/xtal/site.hpp

```diff
@@ -43,6 +44,9 @@ class Site
     CASM::xtal::Site casm_site;
 };
 
+// Applying SymOp onto site by doing an * operator overload
+Site operator*(const sym::CartOp& sym_op, const Site& site);
```

Comment:
See comment for `Coordinate`.

Also the documentation isn't super helpful, I can already tell it's a * operator overload by the function signature. Documentation should explain what the function does, not these kinds of implementation/coding details.